lib/windows/windows_icon_generator.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:image/image.dart';
+import 'package:path/path.dart' as path;
+
+import '../abs/icon_generator.dart';
+import '../constants.dart' as constants;
+import '../custom_exceptions.dart';
+import '../utils.dart' as utils;
```

Comment:
Let's stick to not using relative imports

Reply:
fixed in b421894